File: 김현경/[LV2] 영어 끝말잇기.py
def solution(n, words):
    checked = [words[0]]
    
    for i in range(1, len(words)):
        if words[i][0] != words[i - 1][-1] or words[i] in checked:
            return [i % n + 1, i // n + 1]
        else:
            checked.append(words[i])
    return [0, 0]

File: 조성원/[LV2] 영어 끝말잇기.js
function solution(n, words) {
  const wordSet = new Set();

  for (const [index, word] of words.entries()) {
    if (wordSet.has(word) || (index > 0 && word[0] !== words[index - 1].at(-1)))
      return [(index % n) + 1, Math.ceil((index + 1) / n)];
    wordSet.add(word);
  }

  return [0, 0];
}
